c/isgenq.c

```diff
@@ -145,6 +145,120 @@ int isgenqGetExclusiveLock(const QName *qname,
   return rc;
 }
 
+int isgenqGetSharedLockOrFail(const QName *qname,
+                              const RName *rname,
+                              uint8_t scope,
+                              ENQToken *token,
+                              int *reasonCode) {
+
+  QName localQName = *qname;
+  RName localRName = *rname;
+
+#ifdef METTLE
+  __asm(" ISGENQ MF=L" : "DS"(isgenqParmList));
+  isgenqParmList = GLBENQPL;
+#else
+  char isgenqParmList[512];
+#endif
+
+  int rc = 0, rsn = 0;
+  __asm(
+      ASM_PREFIX
+      "         PUSH USING                                                     \n"
+      "         DROP                                                           \n"
+      "ENQOBTS  LARL  2,ENQOBTS                                                \n"
+      "         USING ENQOBTS,2                                                \n"
+      "         ISGENQ   REQUEST=OBTAIN"
+      ",TEST=NO"
```

Comment:
I understand that starting continuations in column 1 is convenient so you don't have to add continuation characters but it's awkward to read.

Reply:
Style issue so not important